pkg/kube/deployments.go

```diff
@@ -187,7 +194,15 @@ func NewProxyStatefulSet(image types.ImageDetails, serviceInterface types.Servic
 									Value: "json",
 								},
 								{
-									Name: "NAMESPACE",
+									Name:  "APPLICATION_NAME",
```

Comment:
Why do we need the APPLICATION_NAME? (Not saying we don't, just clarifying where it is needed).

Reply:
Agree it is not needed and will remove. It is not a recommended label and it was simply carried over from looking at the transport deployment.